src/cli/List.cpp

```diff
@@ -57,7 +58,7 @@ int List::execute(int argc, char** argv)
     parser.process(arguments);
 
     const QStringList args = parser.positionalArguments();
-    if (args.size() != 1) {
+    if (args.size() != 1 && args.size() != 2) {
```

Comment:
Your help message shows "Default is /" but you are making the group argument mandatory. Is this intentional?